src/gui/dialogues/newflightdialog.cpp

```diff
@@ -481,9 +485,9 @@ void NewFlightDialog::fillDeductibleData()
  * no input validation is done in this step and input data is assumed to be valid.
  * \return
  */
-TableData NewFlightDialog::collectInput()
+RowData NewFlightDialog::collectInput()
 {
-    TableData newData;
+    RowData newData;
     DEB("Collecting Input...");
```

Comment:
throwaway comment.
We can define DEB as
```c++
#define DEB qDebug() << ___PRETTY_FUNCTION___
DEB << "..." << "asdf";
```
we just shorten the akward typing of qDebug() to DEB no need for parentheses